### PR TITLE
Update UG Command Format

### PR DESCRIPTION
**Pull request by zzhikai**
https://zzhikai.github.io/tp/UserGuide.html
Changes
- Update Notes on Command Format
- Update UI picture
- Update caution statement inside edit command

Things to note:

- To add UI and annotation

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -71,23 +71,37 @@ Cautions are placed in this guide to serve as warnings for certain actions.
...
-
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [s/SKILLS]` can be used as `n/John Doe s/Python` or as `n/John Doe`.
+* The command word and the item type for the command are case-sensitive. They have to be `lower_case`. <br>
+  e.g. in `edit 1 n/NAME`, the command word `add` is valid as it is in
+  `lower_case` and `n/` has to be `lower_case` as well.<br>
+  That is, `Add n/John Doe...` will be an invalid command because the command word `Add` is not `lower_case`.
+
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. in `add n/NAME p/PHONE_NUMBER e/EMAIL j/JOB r/ROUND [s/SKILL]...`.<br>
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:80]: Seems a bit long, probably can just do `sort f/NAME ...` or `edit` as a shorter example to cut length as the features probably have examples as well? Same for other points.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -71,23 +71,37 @@ Cautions are placed in this guide to serve as warnings for certain actions.
...
+  `NAME`, `PHONE_NUMBER`, `EMAIL`, `JOB`, `ROUND`, `SKILL` are parameters which can be used 
+  as<br>
+  `add n/John Doe p/99998888 e/johndoe@email.com j/Software Engineer r/Interview s/Python`. 
+
+* Items in square brackets are optional. These parameters are optional because they are additional information you may
+  not have yet.<br>
+  e.g `add n/John Doe p/99998888 e/johndoe@email.com j/Software Engineer r/Interview s/Python` can be used as<br>
+  `add n/John Doe p/99998888 e/johndoe@email.com j/Software Engineer r/Interview` without `s/Python`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:90]: Probably can combine this point with the previous point. I believe only Skills is optional and allow multiple inputs. Can help to cut down length.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -71,23 +71,37 @@ Cautions are placed in this guide to serve as warnings for certain actions.
...
+  `lower_case` and `n/` has to be `lower_case` as well.<br>
+  That is, `Add n/John Doe...` will be an invalid command because the command word `Add` is not `lower_case`.
+
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. in `add n/NAME p/PHONE_NUMBER e/EMAIL j/JOB r/ROUND [s/SKILL]...`.<br> 
+  `NAME`, `PHONE_NUMBER`, `EMAIL`, `JOB`, `ROUND`, `SKILL` are parameters which can be used 
+  as<br>
+  `add n/John Doe p/99998888 e/johndoe@email.com j/Software Engineer r/Interview s/Python`. 
+
+* Items in square brackets are optional. These parameters are optional because they are additional information you may
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:85]: Parameters might be too technical as mentioned by Prof Dara. Maybe input instead? Need to change for other parts in this Notes box

File: docs/UserGuide.md [hunk 4]
```diff
@@ -290,6 +304,11 @@ edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [j/JOB] [r/ROUND] [s/SKILL]...
   e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
 * Only positive indexes are edited. <br>
   e.g As we label our applicants starting from `1...`, an index of `-1` will not be tagged to an applicant.
+* You can remove all of an applicant's skills by simply typing `edit INDEX s/` without adding anything behind `s/` to
```
Comment (RyanCheungJF) [inline docs/UserGuide.md:307]: Don't think it needs the last 4 words

```suggestion
* You can remove all of an applicant's skills by simply typing `edit INDEX s/` without adding anything behind `s/`.
```

**Review by RyanCheungJF — COMMENTED** [review]
Looks good, if could find ways to shorten the Notes box and use less technical terms, that would be great!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -71,23 +71,37 @@ Cautions are placed in this guide to serve as warnings for certain actions.
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [s/SKILLS]` can be used as `n/John Doe s/Python` or as `n/John Doe`.
+* The command word and the item type for the command are case-sensitive. They have to be `lower_case`. <br>
```
[the remark this answers is not in the document]
Comment (ryantanlien) [reply docs/UserGuide.md:74]: Agreed, prefix would be a better word.

File: docs/UserGuide.md (hunk 3 above)
Comment (ryantanlien) [reply docs/UserGuide.md:85]: parameters to input sounds good.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -71,23 +71,37 @@ Cautions are placed in this guide to serve as warnings for certain actions.
...
+  not have yet.<br>
+  e.g `add n/John Doe p/99998888 e/johndoe@email.com j/Software Engineer r/Interview s/Python` can be used as<br>
+  `add n/John Doe p/99998888 e/johndoe@email.com j/Software Engineer r/Interview` without `s/Python`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[s/SKILLS]…​` can be used as (i.e. 0 times), `s/Python`, `s/Python s/Java` etc.
+  e.g. `[s/SKILL]…​` can be used as `s/Python`, `s/Python s/Java` etc.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```
Comment (ryantanlien) [inline docs/UserGuide.md:94]: Be consistent with wording. Here the examples are should be named as prefixes. Maybe have a definition at the top for what is a prefix. "Example: From here on, any input consisting a letter followed by a slash will be labelled as a prefix. e.g n/"

**Review by ryantanlien — CHANGES_REQUESTED** [review]

**Review by ryantanlien — APPROVED** [review]
LGTM!

**Pull request merged**